tests/unit/frameworks/test_langchain.py

```diff
@@ -99,6 +99,7 @@ def _create_effect(**kwargs: str) -> MagicMock:
 def test_run_langchain_agent_custom_args() -> None:
     create_mock = MagicMock()
     agent_mock = AsyncMock()
+    agent_mock.ainvoke.return_value = MagicMock()
```

Comment:
TIL, I didn't know about `ainvoke`

Reply:
Since it's a mock, you can attach any attribute, I could have done `agent_mock.blah-blah.return_value="me"`. The ainvoke matches up with the langchain agent.ainvoke call